pytorch_lightning/trainer/trainer.py

```diff
@@ -319,10 +313,10 @@ def __init__(
         # init callbacks
         # Declare attributes to be set in callback_connector on_trainer_init
         self.checkpoint_callback: Union[ModelCheckpoint, bool] = checkpoint_callback
-        self.early_stop_callback: Optional[Union[EarlyStopping, bool]] = early_stop_callback
+        self.early_stop_callback: Optional[Union[EarlyStopping, bool]] = False
```

Comment:
why not remove this altogether? and the code for initializing early_stop in on_trainer_init?